opnode/flags/p2p_flags.go

```diff
@@ -147,7 +147,7 @@ var (
 		Usage:    "User-agent string to share via LibP2P identify. If empty it defaults to 'optimism-VERSIONHERE'.",
```

Comment:
Leave todo to add version later? Could also cut `-VERSIONHERE`

Reply:
removed the `-VERSIONHERE` part. Think we might want to use the new `runtime/debug` package to grab a version, or continue the current makefile + var override thing.